### PR TITLE
Add TextEncoder and TextDecoder to the module 'util' (Fixes #6326)

### LINKED ISSUE #6326: TextDecoder and TextEncoder missing from node's "util" module
Flow has a module definition for `util`:
https://github.com/facebook/flow/blob/4d8376ab1541c7ed70fa57b243bfe377609cd801/lib/node.js#L1711-L1723

But it lacks the definition for TextDecoder and TextEncoder that node has since v8.3.

Code to reproduce:
```javascript
import { TextEncoder, TextDecoder } from 'util';
```
Error:
```
Cannot import TextDecoder because there is no TextDecoder export in util.
Cannot import TextEncoder because there is no TextEncoder export in util.
```

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

File: lib/node.js [hunk 1]
```diff
@@ -1720,6 +1720,8 @@ declare module "util" {
   declare function inherits(constructor: Function, superConstructor: Function): void;
   declare function deprecate(f: Function, string: string): Function;
   declare function promisify(f: Function): Function;
+  declare export var TextEncoder: Class<TextEncoder>;
+  declare export var TextDecoder: Class<TextDecoder>;
```
Comment (julienw) [inline lib/node.js]: If I don't use the `export` (like for others module), flow doesn't find `TextEncoder`... but finds `TextDecoder`, for my test file. Super strange.

File: tests/node_tests/node_tests.exp [hunk 2]
```diff
@@ -1334,8 +1334,8 @@ References:
    process/nextTick.js:27:3
```
Comment (julienw) [inline tests/node_tests/node_tests.exp]: only line number changes

File: tests/node_tests/util/text-utils.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { TextDecoder, TextEncoder } from 'util';
```
Comment (julienw) [inline tests/node_tests/util/text-utils.js]: I manually checked that when I use `require` this works as well.

**Comment by julienw** [comment]
Please do not review for now, I noticed I broke other exports for 'util'.

**Comment by julienw** [comment]
OK, should be fixed

File: lib/node.js [hunk 4]
```diff
@@ -1709,17 +1709,19 @@ type util$InspectOptions = {
...
-  declare function format(format: string, ...placeholders: any): string;
-  declare function log(string: string): void;
-  declare function inspect(object: any, options?: util$InspectOptions): string;
-  declare function isArray(object: any): boolean;
-  declare function isRegExp(object: any): boolean;
-  declare function isDate(object: any): boolean;
-  declare function isError(object: any): boolean;
-  declare function inherits(constructor: Function, superConstructor: Function): void;
-  declare function deprecate(f: Function, string: string): Function;
-  declare function promisify(f: Function): Function;
```
Comment (julienw) [inline lib/node.js]: TBH I don't know hiw this could work before :/

**Comment by nmote** [comment]
Looks like the CLA was never signed, feel free to reopen if this changes.

**Pull request closed without merging**